bands/models.py

```diff
@@ -18,81 +18,106 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info_id = Column(UUID, ForeignKey('band_info.id'), nullable=True)
+    bio_id = Column(UUID, ForeignKey('band_bio.id'), nullable=True)
+    taxes_id = Column(UUID, ForeignKey('band_taxes.id'), nullable=True)
+    stageplot_id = Column(UUID, ForeignKey('band_stage_plot.id'), nullable=True)
+    panel_id = Column(UUID, ForeignKey('band_panel.id'), nullable=True)
+    merch_id = Column(UUID, ForeignKey('band_merch.id'), nullable=True)
+    charity_id = Column(UUID, ForeignKey('band_charity.id'), nullable=True)
```

Comment:
I would strongly advise you to reverse the direction of the foreign keys here.  In other words, make ``BandInfo`` have a ``.band_id`` foreign key instead of the reverse.

Presumably you wanted to be able to say something like ``if band.info_id`` in order to determine whether or not the band has filled out their info.  But if you set up a ``backref`` on the ``relationship`` you can still just say ``if band.info`` to accomplish the same thing.  This is a lot more idiomatic, because having nullable foreign keys is something we generally want to avoid (except in some specific circumstances like Attendees/Groups).